whipper/common/config.py

```diff
@@ -72,6 +74,13 @@ def get(self, section, option):
     def getboolean(self, section, option):
         return self._getter('boolean', section, option)
 
+    # musicbrainz section
+
+    def get_musicbrainz_server(self):
+        server = self.get('musicbrainz', 'server') or 'musicbrainz.org'
+        server_url = server if re.match(r'^\D*//', server) else '//' + server
+        return urlparse(server_url).netloc
+
```

Comment:
Maybe this is an alternative without regex?

```
>>> server
'musicbrainz.org'
>>> server_url = urlparse(server)
>>> if server_url.scheme == '': server_url = urlparse('//' + server)
...
>>> server_url.netloc
'musicbrainz.org'
```

To be fair, I'd even prefer it to accept only `scheme == ''` or `scheme in ('http', 'https')`.

This way, afaik, people can set the server to 'ftp://musicbrainz.org' and have it still work.

Reply:
Or even only accept no scheme only, since the documentation states:

``` the MusicBrainz server to connect to, in `host:[port]` format. Defaults to `musicbrainz.org`.```

Reply:
For the record, you don't have to change it, but we should update the documentation to reflect what we actually accept in any case. Otherwise it looks great!

Reply:
+1 for implementation without using the regular expression, I like the no-scheme suggestion the best.

Reply:
@MerlijnWajer The alternative would be cleaner, but it doesn't quite work in some cases:

```
>>> urlparse('192.168.2.97/hello').scheme
192.168.2.97
```

Reply:
`192.168.2.97/hello` is indeed not what the docs specify. If you can only set the hostname I would simply not accept `192.168.2.97/hello` as valid URL. If anything, it'll be confusing to people who do set the URI and expect it to have any effect.